### ISSUE TITLE
MDpocket pocket characterization segmentation fault

### ISSUE THREAD

**Issue by obaedaakkad** [Discngine/fpocket#74]
Hi,

I have a problem running the pocket characterization mode of MDpocket with the conda version on Ubuntu 18.04. on every trajectory and selected pocket i get the same error message after running:

mdpocket --trajectory_file nsp1_pock_r0.xtc --trajectory_format xtc -f nsp1_ref.pdb --selected_pocket pocketB_r1_iso_0.5.pdb 

I get:

Format of trajectory xtc nsp1_pock_r0.xtc 0
QH6075 qhull input error: short input file.  Did not find dimension and number of points
While executing:  | qvoronoi p i Pp Qz Qt
Options selected for Qhull 2015.2 2016/01/18:
  run-id 891274485  voronoi  _bbound-last  _coplanar-keep  points  incidence
  Pprecision-ignore  Qz-infinity-point  Qtriangulate
Segmentation fault (core dumped)

Thank you for your help !

**Comment by pschmidtke** [comment]
Is there any chance you can share minimum failing example?

**Comment by obaedaakkad** [comment]
yes, do you want me to send you the files i used for the analyzis ?

**Comment by obaedaakkad** [comment]
Here are the files i used for one of my mdpocket characterization

https://we.tl/t-ZB1nGJiHmL

**Comment by Dan-Burns** [comment]
I'm having the same problem. I compiled fpocket and have tried using xtc inputs as well as a list of pdbs. My trajectory is aligned. I'm on Ubuntu 24 and on a capable workstation. 

I figured the pdb list should work no matter what but for the trajectories, if it matters, I'm using xtc files generated with MDAnalysis from openMM dcd files. I've used files with only the protein as well as the fully solvated system.
